contracts/contracts/vault/OETHZapper.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { IOUSD } from "../interfaces/IOUSD.sol";
+import { IVault } from "../interfaces/IVault.sol";
+import { IWETH9 } from "../interfaces/IWETH9.sol";
+import { ISfrxETH } from "../interfaces/ISfrxETH.sol";
+
+contract OETHZapper {
+    IOUSD immutable oeth;
+    IVault immutable vault;
+    IWETH9 constant weth = IWETH9(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2);
+    ISfrxETH constant sfrxeth =
+        ISfrxETH(0xac3E018457B222d93114458476f3E3416Abbe38F);
+    address constant ETH_MARKER = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
+    address constant FRXETH = 0x5E8422345238F34275888049021821E8E08CAa1f;
```

Comment:
confirmed all these 4 addresses above are correct... well except for ETH_MARKER ofc